### PR TITLE
docs(readme): added info about adding Tegel to stencil.js project

### PR DESCRIPTION
**Pull request by timrombergjakobsson**
**Describe pull-request**  
Added description on how to use Tegel components in a stencil.js project. 

**Solving issue**  
Fixes: [DTS-1490](https://tegel.atlassian.net/jira/software/projects/DTS/boards/1?assignee=63ff6d6c314f5088138187f0&selectedIssue=DTS-1490)

**How to test**  
1. Go to Intro -> Tegel Beta Installation
2. Check in last section at the end and possibly try it out locally by creating a stencil.js project and installing Tegel. 

[DTS-1490]: https://tegel.atlassian.net/browse/DTS-1490?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: tegel/src/stories/Installation/Installation.stories.mdx [hunk 1]
```diff
@@ -134,3 +134,17 @@ export class AppModule { }
...
 ```
 
+## Stencil.js
+
+1. Run `npm install @scania/tegel`
+2. `import '@scania/tegel';` in your stencil component and then you can use your Tegel component of choice like this:
+```ts
+<sdds-button type="primary" size="lg" text="Button"></sdds-button>`
+```
+And don't forget to set ```shadow: false``` for the component.
```
Comment (mJarsater) [inline tegel/src/stories/Installation/Installation.stories.mdx:144]: Does this mean that the users can't use our component in any component were they want to use the shadowDOM?

Comment (timrombergjakobsson) [reply tegel/src/stories/Installation/Installation.stories.mdx:144]: Skärmavbild 2023-04-05 kl  10 25 00

Comment (mJarsater) [reply tegel/src/stories/Installation/Installation.stories.mdx:144]: Alright. So accordion to the text above ideally we should not use shadowDOM to enable people using our components to use the shadowDOM. This seems like a bigger discussion though.

**Review by mJarsater — APPROVED** [review]
✅

File: tegel/src/stories/Installation/Installation.stories.mdx (hunk 1 above)
Comment (timrombergjakobsson) [reply tegel/src/stories/Installation/Installation.stories.mdx:144]: @mJarsater maybe something that can be addressed with the slot API?

**Pull request merged**